githooks/prepare-commit-msg

```diff
@@ -96,6 +96,23 @@ ${cchar}Release note (bug fix): \\
 "
 fi
 
+# Add an issue or epic reference.
+if ! grep -q -E '^(?i:close[sd]?|fix(?:e[sd])?|resolve[sd]?)' "$1"; then
+	sed_script+="/$cchar Please enter the commit message for your changes./i\\
+$cchar Please enter a valid issue or epic reference:\\
+${cchar}Fixes: #77376\\
+${cchar}Part of https://cockroachlabs.atlassian.net/browse/CRDB-13558\\
```

Comment:
Suggesting these changes because issue references normally shouldn't be to CRDB issues because the cockroachdb/cockroach issues are the source of truth and it's easier to digest these suggestions if their formatting is the same.

```suggestion
${cchar}Part of: https://cockroachlabs.atlassian.net/browse/DOC-1355\\
```